src/main/scala/uk/gov/hmrc/cache/repository/CacheRepository.scala

```diff
@@ -37,12 +36,6 @@ trait CacheRepository extends ReactiveRepository[Cache, Id] with UniqueIndexViol
   protected def saveOrUpdate(findQuery: => Future[Option[Cache]], ifNotFound: => Future[Cache], modifiers: (Cache) => Cache)(implicit ec: ExecutionContext): Future[DatabaseUpdate[Cache]] = ???
 }
 
-@deprecated("Please use injected CacheMongoRepository to your class", since = "6.x")
-object CacheRepository extends MongoDbConnection {
```

Comment:
`MongoDbConnection` was dropped from simple-reactivemongo since it required a static Play application (which is no longer available in Play 2.8)